OpenRA.Mods.Common/Widgets/Logic/Ingame/MenuButtonsChromeLogic.cs

```diff
@@ -85,8 +94,28 @@ public MenuButtonsChromeLogic(Widget widget, World world)
 			if (stats != null)
 			{
 				stats.IsDisabled = () => disableSystemButtons || world.Map.Visibility.HasFlag(MapVisibility.MissionSelector);
-				stats.OnClick = () => OpenMenuPanel(stats);
+				stats.OnClick = () => OpenMenuPanel(stats, new WidgetArgs() { { "activePanel", ObserverStatsPanel.Basic } });
 			}
+
+			var keyListener = widget.GetOrNull<LogicKeyListenerWidget>("OBSERVER_KEY_LISTENER");
+			if (keyListener != null)
```

Comment:
Style nit: if a single statement includes its own inner blocks, then the outer block should have braces to avoid ambiguity.